### PR TITLE
change port to 3000

### PR DESCRIPTION
**Pull request by chiachil**
廖家淇

### CODE REVIEW

**Comment by cwpeng** [comment]
Please add rule to `main` branch: require at least 1 review approval before merging. You will see the changes in this page if you finished setup it.

**Comment by chiachil** [comment]
Hi, I've added rule to the main branch, and it looks like I need your approval before I press the merge button. Hope it works this time. THX

**Review by cwpeng — APPROVED** [review]
OK.

**Pull request merged**
